lib/honeybadger.ex

```diff
@@ -14,6 +14,7 @@ defmodule Honeybadger do
       config :honeybadger,
         api_key: "mysupersecretkey",
         environment_name: :prod,
+        revision: "710a4d9bb5afdf31c156d042c7f48f3c107d30e1",
```

Comment:
This isn't something we'd expect people to add manually, so I don't think be listed as an example.